scripts/utils/zowe-install-xmem.sh

```diff
@@ -1,32 +1,27 @@
 #!/bin/sh
-################################################################################
-# This program and the accompanying materials are made available under the terms of the
-# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
 # https://www.eclipse.org/legal/epl-v20.html
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-# Copyright IBM Corporation 2019, 2020
-################################################################################
+# Copyright Contributors to the Zowe Project. 2019, 2020
```

Comment:
Steve says the IBM copyright is the right one.

Reply:
I was told all Zowe contributions must assign copyright to "Contributors to the Zowe Project", as it's not an IBM open source project.

Reply:
I did the requested update, and added the fix for #1550